<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWoMainWindow</class>
 <widget class="QMainWindow" name="QWoMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWoMainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qwoterm.qrc">
    <normaloff>:/woterm/resource/skin/woterm4.png</normaloff>:/woterm/resource/skin/woterm4.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSessionNew"/>
    <addaction name="actionSessionManage"/>
    <addaction name="separator"/>
    <addaction name="actionRestore"/>
    <addaction name="actionBackup"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionToolBar"/>
    <addaction name="actionSessionList"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionOption"/>
    <addaction name="actionIdentityManage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWetsite"/>
    <addaction name="actionDocument"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="text">
    <string>Tool Bar</string>
   </property>
  </action>
  <action name="actionSessionList">
   <property name="text">
    <string>Session List</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="text">
    <string>Option</string>
   </property>
  </action>
  <action name="actionSessionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSessionManage">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionIdentityManage">
   <property name="text">
    <string>Identity Manage</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDocument">
   <property name="text">
    <string>Online Document</string>
   </property>
  </action>
  <action name="actionWetsite">
   <property name="text">
    <string>Office Wetsite</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="text">
    <string>Backup</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qwoterm.qrc"/>
 </resources>
 <connections/>
</ui>
